rir/src/compiler/pir_tests.cpp

```diff
@@ -49,7 +49,8 @@ SEXP compileToRir(const std::string& context, const std::string& expr, SEXP supe
 
 std::unordered_map<std::string, pir::Closure*>
 compileRir2Pir(SEXP env, pir::Module* m){
-    pir::Rir2PirCompiler cmp(m, pir::DebugOptions() | pir::DebugFlag::PrintFinalPir);
+    pir::StreamLogger logger((pir::DebugOptions()));
```

Comment:
there is some parser ambiguity without them. I am not sure I understand why, but they are needed.

Reply:
on the other hand I added the PrintFinalPir flag again, since that runs the printing code, which sometimes revealed bugs in the past. and with this re-added the () are not needed anymore :)

Reply:
Sounds like the [most vexing parse](https://en.wikipedia.org/wiki/Most_vexing_parse). Braces instead of parens would also have fixed it.